bukkit/src/main/java/de/maxhenkel/voicechat/config/ServerConfig.java

```diff
@@ -30,79 +30,79 @@ public ServerConfig(ConfigBuilder builder) {
         voiceChatPort = builder
                 .integerEntry("port", 24454, -1, 65535,
                         "The port of the voice chat server",
-                        "Setting this to \"-1\" sets the port to the Minecraft servers port (Not recommended)"
+                        "Setting this to -1 sets the voice chat port to the Minecraft server port (Not recommended)"
```

Comment:
This should be left in quotes to be precise

Reply:
Okay, I'll readd all the double quotes I removed in the configuration file comments, and replace them with single quotes. Does that work for you?

Reply:
Yep, thats good